lib/registries/addon/branded/discover/template.hbs

```diff
@@ -7,6 +7,7 @@
     <SearchPage
         @provider={{this.model}}
         @defaultQueryOptions={{this.defaultQueryOptions}}
+        @resourceType={{'Registration,RegistrationComponent'}}
```

Comment:
If only we can import `enum`s in the template...

Reply:
Well, hypothetically we could if we wanted to try our first polaris component... ⭐